### PR TITLE
ENG-8282 make keycloak default authentication (K8s generic Platforms)

### PR DESCRIPTION
**Pull request by ommkar-xl**

### CODE REVIEW

File: templates/xl-release.yaml [hunk 1]
```diff
@@ -100,6 +100,7 @@ spec:
             failureThreshold: {{ .Values.HealthProbeFailureThreshold }}
           {{- end }}
           env:
+
```
Comment (rprasad-digitalai) [inline templates/xl-release.yaml:103]: Do we really need this change?

File: values-nginx.yaml [hunk 2]
```diff
@@ -50,13 +128,13 @@ ingress:
...
     kubernetes.io/ingress.class: nginx
     nginx.ingress.kubernetes.io/ssl-redirect: "false"
-    nginx.ingress.kubernetes.io/rewrite-target: /$2
+    nginx.ingress.kubernetes.io/rewrite-target: /
     nginx.ingress.kubernetes.io/affinity: cookie
     nginx.ingress.kubernetes.io/session-cookie-name: JSESSIONID
     nginx.ingress.kubernetes.io/proxy-connect-timeout: "60"
     nginx.ingress.kubernetes.io/proxy-read-timeout: "60"
     nginx.ingress.kubernetes.io/proxy-send-timeout: "60"
-  path: /xl-release(/|$)(.*)
```
Comment (rprasad-digitalai) [inline values-nginx.yaml:59]: Are we removing path based routing?

File: templates/xl-release.yaml (hunk 1 above)
Comment (ommkar-xl) [reply templates/xl-release.yaml]: Removed the extra space

File: values-nginx.yaml (hunk 2 above)
Comment (ommkar-xl) [reply values-nginx.yaml:59]: For Keycloak, yes. After the "path" issue is fixed will update the ingress resource.

**Review by rprasad-digitalai — APPROVED** [review]
Looks good to me

**Review by sagarkar-xl — APPROVED** [review]

**Pull request merged**